### ISSUE TITLE
[Bug] SHORT DESCRIPTION

### ISSUE THREAD

**Issue by valenipes** [NeoApplications/Neo-Backup#849]
**Description**
Whatsapp backup of /storage/emulated/10/Android/media/com.whatsapp not working

**Steps To Reproduce**

1. I Installed Whatsapp and NeoBackup
2. I restored a previous backup from Whatsapp, done manually, not with NeoBackup (I simply copied and pasted the old /Android folder replacing the new one)
3. I tried to backup all Whatsapp data (only excluding APK) without compression, but files in media folder weren't backed up, because the original size was around 200MB, and the backup size around 20MB
4. To check point 3. , I finally restored the backup, and files, photo, audio... (contents of /media/com.whatsapp) weren't backed up

**Expected behavior**
I'd like to restore also the media folder.

**System Information(please complete the following information):**

- Device: POCO X3 NFC
- Android Version: 13
- ROM: LOS
- App's Version: 8.3.6

-----------------------------

**Additional Notes**

* If you're on the latest Test version(from the TG-Channel or Matrix-Room) please report there directly, not here.

* If you're on the latest github version:
  
  * Please read the [FAQ](https://github.com/NeoApplications/Neo-Backup/blob/master/FAQ.md) first.
  * Please read all the open Issues. If one does match yours, leave a comment there if there's some extra information you can provide.

* If you're not on the latest github version. Update, then make sure that the bug is still there. If it's go further with reporting.

Thanks for reporting.

**Comment by machiav3lli** [comment]
The user ID points out to a work profile, so maybe that's the culprit… so can you please confirm that we're talking about a work profile app. If so, I would like to point out that work profile support is still on the todo list.

**Comment by valenipes** [comment]
Yes I confirm, whatsapp is in the secondary user profile. I'm sorry, I didn't know work profile wasn't supported. Anyway, I noticed that some data are backed up, idk if this may help.

**Comment by hg42** [comment]
> 1. I Installed Whatsapp and NeoBackup

I assume both were installed in the work profile, right?

There were certain problems with work profiles (Android changes etc.).
A bunch of them were fixed lately.
For a short time I used it myself, so I could test it in real life.
So, you might try again.

Note, that the simplest way is to use a backup directory in the internal storage of the work profile.
Other locations might be more complex.
It **is** possible to use it with external storage, but only with latest pumpkin apks (see telegram group) and shadowRoot.
For such things ask there... (if you don't use Telegram, ask here)
